src/react/forms.jsx

```diff
@@ -5,9 +5,28 @@
 
 'use strict';
 
+var formControlClass = 'mui-form-control',
+    formGroupClass = 'mui-form-group',
+    floatingLabelBaseClass = 'mui-form-floating-label',
+    floatingLabelActiveClass = floatingLabelBaseClass + '-active',
+    animationName = 'mui-form-floating-label-inserted',
+    _supportsPointerEvents;
 
-var jqLite = require('../js/lib/jqLite.js'),
-    forms = require('../js/forms.js');
+var React = require('react');
+var cx = require('classnames');
+
+/**
+ * Check if client supports pointer events.
+ */
+function supportsPointerEvents() {
```

Comment:
yeah, I know, but I wan't to keep the code contained in jsx module files, so you can use what you need by require('whateverWidget.jsx'); And it doesn't need anything else.